src/Workleap.DomainEventPropagation.Abstractions/IEventPropagationClient.cs

```diff
@@ -2,15 +2,9 @@ namespace Workleap.DomainEventPropagation;
 
 public interface IEventPropagationClient
 {
-    Task PublishDomainEventAsync<T>(string subject, T domainEvent, CancellationToken cancellationToken)
```

Comment:
Don't think we can actually do that, signatures where a subject is specified as string are used within OV's codebase